build.gradle

```diff
@@ -313,6 +313,12 @@ project(':core') {
     from sourceSets.test.output
   }
 
+  task publishDocs(type: Jar) {
```

Comment:
Oh, then I agree with Gwen, it's odd to upload that to Maven as a jar. Sorry for the noise.